### PR TITLE
Better error printing from the rebar plugin

### PR DESCRIPTION
**Pull request by joedevivo**
The problem is that cuttlefish assumes that lager is running for its logging.

That's always true, except when running the rebar plugin. Rebar doesn't have lager built in, and I'm not saying that it should. What this PR does is io:format errors IF errors are returned to the rebar_plugin. Any other path assumes they've been lager'd.

Also piggybacked in here a fix for vm_args that are being output as ~s even when they're integers. This is probably the little devil behind #48. In fact, I know it is because, spoiler alert, I've got a fix for that coming too.

### CODE REVIEW

File: src/cuttlefish_rebar_plugin.erl [hunk 1]
```diff
@@ -50,18 +50,14 @@ generate(Config0, ReltoolFile) ->
 
                     io:format("Schema: ~p~n", [Schemas]),
 
-                    {_Translations, Mappings, _Validators} = cuttlefish_schema:files(Schemas),
+                    case cuttlefish_schema:files(Schemas) of 
+                        {error, Es} ->
+                            io:format("Error loading schmeas: ~n"),
```
Comment (seancribbs) [inline src/cuttlefish_rebar_plugin.erl]: typo'd

Comment (joedevivo) [inline src/cuttlefish_rebar_plugin.erl]: I was referring to Captain Hook's first mate.

Comment (joedevivo) [inline src/cuttlefish_rebar_plugin.erl]: fixed.

**Comment by joedevivo** [comment]
For the record, the title of this PR is "better" printing, not "best" printing. You have to agree it's better than the zero printing it did before :)

**Comment by seancribbs** [comment]
:+1: much improve, wow

**Comment by joedevivo** [comment]
o man o man o man

**Pull request merged**